.circleci/config.yml

```
@@ -177,7 +177,7 @@ jobs:
     steps:
       - cabal_build_and_test:
           project_file: "cabal.ghc9.project"
-          allow_test_failures: true
+          extra_test_flags: ' --test-options '' -p "\$0 != \"Tests.Benchmarks.text.Data/Text/Foreign.hs\" && \$0 !~ /Tests.Micro.typeclass-pos./ " '' '
```

```suggestion
          extra_test_flags: ' --test-options '' -p "\$0 != \"Tests.Benchmarks.text.Data/Text/Foreign.hs\" && ! /Tests.Micro.typeclass-pos./ " '' '
```
Syntax and escapes are so hard ...